README.md

```diff
@@ -1,4 +1,4 @@
-## Hi, I am Kaiwalya! Great to see you here! 👋
+# ![Typing SVG](https://readme-typing-svg.demolab.com?font=Fira+Code&size=30&pause=500&vCenter=true&width=800&color=A607FC&lines=Hey+there,+I+am+Kaiwalya!%F0%9F%91%8B;Great+to+see+you+here!+%F0%9F%99%8F;)
```

Comment:
Hey @giteshsarvaiya thank you so much for your contribution but I would like it to be static. Please feel free to contribute in other parts of the readme :)